packages/fx-core/resource/package.nls.json

```diff
@@ -406,6 +406,8 @@
   "core.common.ResolveServerUrlFailed": "Unable to resolve the server URL: please make sure that the environment variable %s is defined.",
   "core.common.OperationOnlyContainsOptionalParam": "Operation %s contains multiple optional parameters. The first optional parameter is used for this command.",
   "core.common.ConvertSwaggerToOpenAPI": "The Swagger 2.0 file has been converted to OpenAPI 3.0.",
+  "core.common.ErrorFetchApiSpec": "Please make sure your OpenAPI description document can be accessed without authentication or you can download a copy of the document to your local disk and upload the local file.",
```

Comment:
Your OpenAPI specification should be accessible without authentication, otherwise download and start from a local copy.

Reply:
Updated.